docs/implementation-references/getting-started-guide.md

```diff
@@ -0,0 +1,298 @@
+***NOTE***: This is a preliminary implementation of the [FIDO Device Onboard Spec](https://fidoalliance.org/specs/FDO/fido-device-onboard-v1.0-ps-20210323/) published by the FIDO Alliance.
+The implementation is experimental and incomplete, and is not ready for use in any production capacity. Some cryptographic algorithms and encoding formats have
+not been implemented, and any aspect of this implementation is subject to change.
+
+# Getting Started Guide
+
+Quick walk through the E2E flow, so you can get started. Included in this guide:
+
+- [Building FDO PRI Source](#building-fdo-pri-source)
+- [Starting FDO Service Containers](#starting-fdo-service-containers)
+- [Running E2E for PRI device](#running-e2e-for-pri-device)
+- [Building Client-SDK Source](#building-client-sdk-source)
+- [Running E2E for Client-SDK device](#running-e2e-for-client-sdk-device)
+- [Enabling ServiceInfo](#enabling-serviceinfo-transfer)
+- [Keystore Management](#keystore-management)
+
+## Building FDO PRI Source
+
+!!! NOTE
+    For the instructions in this document, `<fdo-pri-src>` refers to the path of the FDO PRI folder 'pri-fidoiot'.
+
+FDO PRI source can be built in two ways:
+
+1. Using the maven build system to build FDO PRI source.
+
+```
+$ cd <fdo-pri-src>
+$ mvn clean install
+```
+
+2.&nbsp;Using docker container to build FDO PRI source
+
+```
+$ cd <fdo-pri-src>/build
+$ sudo docker-compose up --build
```

Comment:
do we need sudo or is it optional?

Reply:
Better to add sudo here..As for most users, docker-compose might not be added to the root group.